website/source/intro/getting-started/first-secret.html.md

```diff
@@ -26,8 +26,30 @@ have the means necessary to decrypt it without Vault.
 
 ## Writing a Secret
 
-Let's start by writing a secret. This is done very simply with the
-`vault write` command, as shown below:
+Let's start by writing a secret. Before we do that though, we need to authenticate with the Vault server. We will be using the root token the server provided us in the previous step:
```

Comment:
Can we have a section before `## Writing a Secret` that goes by something like `## Authenticating the CLI` which details the login procedure? That way this section can explain only about writing a secret and nothing more.